packages/simcore-sdk/requirements/_base.in

```diff
@@ -7,7 +7,7 @@
 --requirement ../../../packages/settings-library/requirements/_base.in
 --requirement ../../../packages/models-library/requirements/_base.in
 
-aiocache
+aiodocker
 aiofiles
```

Comment:
make sure this change in simcore-sdk reqs is also applied downstream (i.e. all services including simcore-sdk).
Be careful though to only do these changes in the requirements downstream though ...  (TIP: ``make reqs upgrade=aiodocker``

Reply:
Thanks for the tip.